src/wdio/services/ServeStaticService.js

```diff
@@ -24,7 +26,16 @@ export default class ServeStaticService {
       await this.stop();
     }
 
-    await ServeStaticService.startService(webpackConfig, port, index, locale, verbose).then((server) => {
+    const serveOptions = {
+      ...site && { site, disk: fse },
```

Comment:
disk is a true false param, are you setting it to a filesystem?

Reply:
I would NEVER do that..... 👀 🙈